scripts/variantstore/variant-annotations-table/GvsCreateVATfromVDS.wdl

```diff
@@ -85,7 +85,7 @@ workflow GvsCreateVATfromVDS {
 
     # If the vat version is undefined or v1 then the vat tables would be named like filter_vat, otherwise filter_vat_v2.
     String effective_vat_version = if (defined(vat_version) && select_first([vat_version]) != "v1") then "_" + select_first([vat_version]) else ""
-    String vat_table_name = filter_set_name + "_vat" + effective_vat_version
+    String effective_vat_table_name = filter_set_name + "_vat" + effective_vat_version
```

Comment:
not "qualified" ? whats the goal of this? maybe we should just pass the name?

Reply:
I needed `vat_table_name` to be an output of the wdl, and couldn't name an internal variable with the same name, so I changed it to `effective_vat_table_name` - using the pattern of a lot of naming nearby.